android/src/reactNativeVersionPatch/nativeHierarchyManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedNativeHierarchyManager.java

```diff
@@ -263,7 +285,13 @@ public synchronized void manageChildren(
     }
 
     // we don't want layout animations in native-stack since it is currently buggy there
+    AnimationsManager animationsManager = ((ReaLayoutAnimator) mReaLayoutAnimator).getAnimationsManager();
```

Comment:
we do we need casting here? If the field is called `mReaLayoutAnimator` why isn't it of type `ReaLayoutAnimator`?

Reply:
Resolve in: https://github.com/software-mansion/react-native-reanimated/pull/4014/commits/12860cc478ab0c9c4eedc371a75cc93d1af79a0f